tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -171,6 +171,17 @@ jobs:
 - template: android-java-api-aar.yml
   parameters:
     buildConfig: 'Release'
+    buildSettings: '$(Build.SourcesDirectory)/tools/ci_build/github/android/default_mobile_aar_build_settings.json'
+    includedOpsConfig: '$(Build.SourcesDirectory)/tools/ci_build/github/android/mobile_package.required_operators.config'
+    artifactName: 'onnxruntime-android-aar'
+    job_name_suffix: 'Mobile'
+
+- template: android-java-api-aar.yml
+  parameters:
+    buildConfig: 'Release'
+    buildSettings: '$(Build.SourcesDirectory)/tools/ci_build/github/android/default_full_aar_build_settings.json'
+    artifactName: 'onnxruntime-nuget-aar'
+    job_name_suffix: 'Full'
```

Comment:
can we use 'onnxruntime-android-full-aar' so when someone is looking at the artifacts listing it's clearer what the directory contains?

Reply:
I used that eariler, changed to nuget-aar to avoid confusion, for example, prevent release manager accidentally release this instead of the mobile package